app/src/main/kotlin/crazyboyfeng/accSettings/acc/AccHandler.kt

```diff
@@ -25,23 +26,27 @@ class AccHandler {
         val accVersionCode = resources.getInteger(R.integer.acc_version_code)
         val accVersionName = resources.getString(R.string.acc_version_name)
         try {
-            cacheAssertFile("acc_v${accVersionName}_.${accVersionCode}.tgz")
-            val installShFile = cacheAssertFile("install-tarball.sh")
-            val command = "sh ${installShFile.absolutePath} acc --non-interactive"
+            cacheAssetFile("acc_v${accVersionName}_${accVersionCode}.tgz")
+            val installShFile = cacheAssetFile("install-tarball.sh")
+            val command = "sh ${installShFile.absolutePath}"
```

Comment:
Not specifying the tarball file location is actually an online installation, not an installation from a bundle.

Not using `--non-interactive` will cause the application to be unresponsive.  
Although using non-blocking installation will not return installation results. However, the user can be informed of the success of the installation by the running status of the application.

Reply:
Oops! We have an issue here.
`--non-interactive` is actually for `install-online.sh`.

The tarball installer is used as follows: `sh install-tarball.sh [module ID] [PARENT installation directory]`
_Module ID_ falls back to `acc`, but I recommend specifying it, since this same installer can be used for other modules as well.
The parent installation directory falls back to `/data/adb/modules` (if Magisk is installed); else, `/data/adb/vr25`. The full path to the module, once installed, ends in `/acc`.

Here's an example: `sh install-tarball.sh acc /data/data/github.vr25.acc/files`
Full path: /data/data/github.vr25.acc/files/acc
When the front-end's data folder is the parent installation directory and Magisk is installed, the installer creates a cleanup script that removes acc and itself on the next boot, after the front-end is uninstalled.

Reply:
It seems I misunderstood with `install-tarball.sh`. Then there is no problem with this change.